dbt/include/snowflake/macros/materializations/dynamic_table/ddl.sql

```diff
@@ -54,7 +54,7 @@
             "database_name",
             "text",
             "target_lag",
-            "warehouse"
+            "warehouse" as "snowflake_warehouse"
```

Comment:
@mikealfare Curious if this raises any warning bells for you, reverting back to `warehouse` in create macro as its the field snowflake expects means we have to somehow change that back to `snowflake_warehouse` on our end. if there is a better way please let me know. currently passing test will run some models in a project to test further.

Reply:
I would do it [here](https://github.com/dbt-labs/dbt-snowflake/blob/2dc3819b36e80898674854e6fce716d2067f1c6d/dbt/adapters/snowflake/relation_configs/dynamic_table.py#L84), the same way we rename `text` to `query`.